packages/reusable-blocks/src/components/reusable-blocks-menu-items/reusable-blocks-manage-button.js

```diff
@@ -44,6 +53,9 @@ function ReusableBlocksManageButton( { clientId } ) {
 			>
 				{ __( 'Manage Reusable blocks' ) }
 			</MenuItem>
+			<MenuItem onClick={ () => convertBlockToStatic( clientId ) }>
+				{ __( 'Convert to Regular Blocks' ) }
```

Comment:
```suggestion
				{ __( 'Convert to regular blocks' ) }
```

Reply:
Thank you for mentioning this! I wasn't aware of this change. 🙂